Configuration_adv.hpp

```diff
@@ -307,63 +307,68 @@
         #endif
         #ifndef AZ_STEPPER_ACCELERATION
-            #define AZ_STEPPER_ACCELERATION 400  // High speeds tend to make these cheap steppers unprecice
+            #define AZ_STEPPER_ACCELERATION 200 * AZ_MICROSTEPPING  // High speeds tend to make these cheap steppers unprecice
         #endif
     #elif AZ_STEPPER_TYPE == STEPPER_TYPE_NEMA17
         #ifndef AZ_STEPPER_SPR
             #define AZ_STEPPER_SPR 400  // NEMA 0.9° = 400  |  NEMA 1.8° = 200
         #endif
         #ifndef AZ_STEPPER_SPEED
-            #define AZ_STEPPER_SPEED 600  // You can change the speed and acceleration of the steppers here. Max. Speed = 3000.
+            #define AZ_STEPPER_SPEED                                                                                                       \
+                100 * AZ_MICROSTEPPING  // You can change the speed and acceleration of the steppers here. Max. Speed = 3000.
         #endif
         #ifndef AZ_STEPPER_ACCELERATION
-            #define AZ_STEPPER_ACCELERATION 1000
+            #define AZ_STEPPER_ACCELERATION 100 * AZ_MICROSTEPPING
         #endif
     #else
         #error Unknown AZ stepper type
     #endif
 
     // the Circumference of the AZ rotation. 808mm dia.
     #define AZ_CIRCUMFERENCE 2538.4f
-    #ifndef AZIMUTH_STEPS_PER_REV
-        #define AZIMUTH_STEPS_PER_REV                                                                                                      \
-            (AZ_CORRECTION_FACTOR * (AZ_CIRCUMFERENCE / (AZ_PULLEY_TEETH * GT2_BELT_PITCH)) * AZ_STEPPER_SPR                               \
-             * AZ_MICROSTEPPING)  // Actually u-steps/rev
-    #endif
+    #define AZIMUTH_STEPS_PER_REV                                                                                                          \
+        (AZ_CORRECTION_FACTOR * (AZ_CIRCUMFERENCE / (AZ_PULLEY_TEETH * GT2_BELT_PITCH)) * AZ_STEPPER_SPR                                   \
+         * AZ_MICROSTEPPING)                                                      // Actually u-steps/rev
     #define AZIMUTH_STEPS_PER_ARC_MINUTE (AZIMUTH_STEPS_PER_REV / (360 * 60.0f))  // Used to determine move distance in steps
 
     // AZ TMC2209 UART settings
     // These settings work only with TMC2209 in UART connection (single wire to TX)
     #if (AZ_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
         #define AZ_RMSCURRENT AZ_MOTOR_CURRENT_RATING *(AZ_OPERATING_CURRENT_SETTING / 100.0f) / 1.414f
 
-        #define AZ_AUDIO_FEEDBACK 0
-
         #define AZ_STALL_VALUE 10  // adjust this value if the RA autohoming sequence often false triggers, or triggers too late
 
         #ifndef USE_VREF
             #define USE_VREF                                                                                                               \
                 0  //By default Vref is ignored when using UART to specify rms current. Only enable if you know what you are doing.
         #endif
+        #ifndef AZ_ALWAYS_ON
```

Comment:
I just did  something similar for the focuser, but wanted the hold current to be a lot  less than the run current. Might it be worth it to use a lower current for holding?

Reply:
What I noticed during testing NEMAs with the alt/az system is that because the movements are so non-continuous (i.e. short pulses from each MAZ/MAL command), unless the steppers stay energized they can't actually make small movements because there isn't enough pressure build up from the motor attempting to turn before it de-energizes again.

In this particular situation I think having the hold current the same as the run current makes sense.